src/video.cpp

```diff
@@ -640,20 +639,18 @@ namespace video {
         { "pic_timing_sei"s, 0 },
         { "max_dec_frame_buffering"s, 1 },
       },
-      // SDR-specific options
-      {
-        { "profile"s, (int) qsv::profile_h264_e::high },
-      },
-      // HDR-specific options
-      {},
-      // Fallback options
+      {},  // SDR-specific options
+      {},  // HDR-specific options
+      {},  // YUV444 SDR-specific options
```

Comment:
QSV doesn't support H.264 High 4:4:4, so it ends up picking H.264 Main profile when the client asks for 4:4:4. We should probably block H.264 4:4:4 from being used with QSV to prevent this.

Reply:
Changed the code to explicitly request the profile QSV doesn't support, so I *think* it should now fail the encoder validation.
In the future (or if QSV still falls back to some valid profile) we might want to add more flexible codec blacklisting logic, but this PR is large enough already so I'd rather not do it here.